modules/nf-core/sourmash/gather/meta.yml

```
@@ -0,0 +1,92 @@
+name: "sourmash_gather"
+description: Search the reference database for the most highly contained genomes using sourmash
+keywords:
+  - hash sketch
+  - signature
+  - kmer
+  - containment
+  - sourmash
+  - genomics
+  - metagenomics
+  - taxonomic classification
+  - taxonomic profiling
+tools:
+  - "sourmash":
+      description: Compute and compare MinHash signatures for DNA data sets.
+      homepage: https://sourmash.readthedocs.io/
+      documentation: https://sourmash.readthedocs.io/
+      tool_dev_url: https://github.com/dib-lab/sourmash
+      doi: "10.1186/s13059-016-0997-x"
```

This is the Mash DOI, which is different than sourmash.
```suggestion
      doi: "10.21105/joss.00027"
```

Can two DOIs be listed? the sourmash tool asks for `10.21105/joss.00027` to be cited when sourmash is used, but gather was published in this preprint: 10.1101/2022.01.11.475838